.github/workflows/reusable-git-issues-jira-automation.yaml

```diff
@@ -56,32 +56,34 @@ jobs:
         if: steps.create-jira-issue.outcome == 'success'
         run: echo "Jira Issue ${{ steps.create-jira-issue.outputs.issue }} was created"
 
-      - name: Label Create Git Issue
-        uses: actions/github-script@v7
-        if: steps.create-jira-issue.outcome == 'success'
-        with:
-          script: |
-            github.rest.issues.addLabels({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              labels: ["${{ steps.create-jira-issue.outputs.issue }}"]
-            })
-
       - name: Add Comment On Jira issue
         if: env.jira_issue_exist == 'true' && (github.event_name == 'issue_comment' || (github.event_name == 'issues' && github.event.action == 'closed'))
         uses: atlassian/gajira-comment@v3
         with:
           issue: ${{ env.akeyless_jira_issue }}
           comment: ${{ env.jira_description }}
 
-      - name: Transition issue
-        if: github.event_name == 'issues' && github.event.action == 'closed' && env.jira_issue_exist == 'true'
-        id: transition
-        uses: atlassian/gajira-transition@v3
-        with:
-          issue: ${{ env.akeyless_jira_issue }}
-          transition: "Done"
+      - name: Prepare Slack Message On Success
+        if: env.jira_issue_exist == 'false' && github.event.action == 'opened'
+        id: slack-message-success-creator
+        run: |
+          SLACK_MESSAGE="${jira_description}
+          *Jira Ticket*: https://akeyless.atlassian.net/browse/${{steps.create-jira-issue.outputs.issue}}"
+          delimiter="$(openssl rand -hex 8)"
+          echo "slack-message<<${delimiter}" >> $GITHUB_ENV
+          echo "${SLACK_MESSAGE//$/%0A}" >> $GITHUB_ENV
+          echo "${delimiter}" >> $GITHUB_ENV
 
-      - name: Issue summary
-        run: echo '### Github issue automation summary -${{ steps.create-jira-issue.outputs.issue }} ${{ env.jira_description }}! :rocket:' >> $GITHUB_STEP_SUMMARY
\ No newline at end of file
+      - name: Slack Success Notification
+        if: env.jira_issue_exist == 'false' && github.event.action == 'opened'
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_CHANNEL: 'operations'
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_ICON: https://github.com/rtCamp.png?size=48
+          SLACK_TITLE: 'New Git Issue Created'
+          SLACK_MESSAGE: '${{ env.slack-message }}'
+          SLACK_USERNAME: githubBot
+          SLACK_WEBHOOK: ${{ secrets.SLACK_OPERATIONS_WEBHOOK }}
+          MSG_MINIMAL: true
+          SLACK_FOOTER: GitHub Issue To Jira Automation
```

Comment:
i think sending slack message to operations is not a good idea, if you still want to send slack messages, please send them to a dedicated channel. Also i would add a failure message for us to understand if we have any issue with this automation